src/base/rss/rss_article.cpp

```diff
@@ -114,16 +115,9 @@ void Article::markAsRead()
 
 QJsonObject Article::toJsonObject() const
 {
-    return {
-        {Str_Id, m_guid},
-        {Str_Date, m_date.toString(Qt::RFC2822Date)},
-        {Str_Title, m_title},
-        {Str_Author, m_author},
-        {Str_Description, m_description},
-        {Str_TorrentURL, m_torrentURL},
-        {Str_Link, m_link},
-        {Str_IsRead, m_isRead}
-    };
+    QJsonObject jsonObj(QJsonObject::fromVariantHash(m_varHash));
```

Comment:
Don't try to cheat it! QJsonObject::fromVariantHash doesn't support all QVariants (at least it doesn't support QDateTime).